### PR TITLE
Event hub trigger

### PR DESCRIPTION
**Pull request by mikhailshilkov**
Add Event Hub triggers for #218

### CODE REVIEW

**Comment by mikhailshilkov** [comment]
tagging @CyrusNajmabadi

File: sdk/nodejs/eventhub/zMixins.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 import * as pulumi from "@pulumi/pulumi";
 import { getServiceBusNamespace, Subscription, Topic } from ".";
+import { EventHub, EventHubConsumerGroup, getEventhubNamespace } from ".";
```
Comment (CyrusNajmabadi) [inline sdk/nodejs/eventhub/zMixins.ts]: the inconsistent casing of EventHub here bothers me :)

Comment (CyrusNajmabadi) [reply sdk/nodejs/eventhub/zMixins.ts]: oh, you can't even fix that.  bleagh.

**Comment by lukehoban** [comment]
Merging in based on @CyrusNajmabadi LGTM.

**Pull request merged**